release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1715,6 +1715,11 @@ In the following tables, features are marked with the following statuses:
 [id="ocp-4-14-deprecated-features"]
 === Deprecated features
 
+[id="ocp-4-14-deprecation-sdn"]
+==== Deprecation of the OpenShift SDN network plugin
+
+OpenShift SDN CNI is deprecated as of {product-title} {product-version}. The network plugin will not be an option for new installations in the next minor release of {product-title}. In a subsequent future release, the OpenShift SDN network plugin will be removed and no longer supported. Red{nbsp}Hat will provide bug fixes and support for this feature until removed, but this feature will no longer receive enhancements. As an alternative to OpenShift SDN CNI, you can use OVN Kubernetes CNI instead.
```

Comment:
As of 4.14 the deprecation is official, so that sentence looks good. But all of the other future removals are still plans that haven't technically happened yet, so I still think that we need to use "planned" for all of these removal plans. Suggestions below, but can feel free to work it in a different way

```suggestion
OpenShift SDN CNI is deprecated as of {product-title} {product-version}. It is currently planned that the network plugin will not be an option for new installations in the next minor release of {product-title}. In a subsequent future release, the OpenShift SDN network plugin is planned to be removed and no longer supported. Red{nbsp}Hat will provide bug fixes and support for this feature until removed, but this feature will no longer receive enhancements. As an alternative to OpenShift SDN CNI, you can use OVN Kubernetes CNI instead.
```